### DISCUSSION TITLE
Scrum Prep Milestone 1 - Week 3

### DISCUSSION THREAD

**Discussion by PedroRodz** [uprm-inso4101-2024-2025-s1/semester-project-regiupr#65] [Scrum Preps]
Weekly Restrospective and Scrum prep
13-20 sept

**Comment by PedroRodz** [comment]
-Attended meeting
-worked on documentation

**Comment by christianmedina-dev** [comment]
Could not work much on the project. Had a hell week

**Comment by carlospepin23** [comment]
-worked on database documentation

**Comment by cristianMarcial** [comment]
finished my assigned task + helped Armando & Jorge on the GUI team tasks
